odoo/orm/models.py

```diff
@@ -6329,6 +6329,9 @@ def __repr__(self):
     def __hash__(self):
         return hash((self._name, frozenset(self._ids)))
 
+    def __deepcopy__(self, memo):
+        return self
```

Comment:
@odoo/rd-framework-py  Without that, you can't use `deepcopy` on a dictionary containing records. Are you ok with that?

Reply:
Actually, as per specs of deepcopy, this should not work, since you cannot `copy` (python function) a recordset. I'm not all against it.  @ryv-odoo what do you think?

Reply:
Fine by me :+1: